certbot-dns-route53/certbot_dns_route53/dns_route53.py

```diff
@@ -11,6 +11,8 @@
 from certbot import interfaces
 from certbot.plugins import dns_common
 
+from acme.magic_typing import DefaultDict, List, Dict # pylint: disable=import-error, unused-import
```

Comment:
Whoops. The `pylint` comment here should be `# pylint: disable=unused-import, no-name-in-module`. 

Happy to explain the "magic" of magic_typing if you're interested.